import/controls/permissionmodel.h

```diff
@@ -0,0 +1,65 @@
+/****************************************************************************
+**
+** Copyright (C) 2020 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef PERMISSIONMODEL_H
+#define PERMISSIONMODEL_H
+
+#include <QAbstractListModel>
+#include "permissionmanager.h"
+
+struct Permission
+{
+    Permission(QString host, QString type, PermissionManager::Capability capability)
+        : m_host(host), m_type(type), m_capability(capability) {}
```

Comment:
```
    Permission(QString host, QString type, PermissionManager::Capability capability)
        : m_host(host)
        , m_type(type)
        , m_capability(capability)
    {
    }
```
In contrast to the splitting parameters of a method over multiple lines. This way it is easier to see when ever somebody adds a new variable to the constructor list.